Cargo.toml

```diff
@@ -32,3 +32,7 @@ once_cell = "1.5"
 default = ["text", "system-fonts"]
 text = ["usvg/text"] # enables SVG Text support
 system-fonts = ["usvg/system-fonts"] # enables system fonts loading
+memmap = ["usvg/memmap"] # enables font loading using memmap
+
+[patch.crates-io]
+fontdb = { path = "fontdb" }
```

Comment:
You should simply replace

```toml
fontdb = { version = "0.6", optional = true, default-features = false }
```

with 

```toml
fontdb = { git = "https://github.com/RazrFalcon/fontdb", rev = "b009936", optional = true, default-features = false }
```

in `usvg/Cargo.toml`. Replace `b009936` with a hash we would get after merging your patch.

`[patch.crates-io]` section should be removed completely.

Reply:
Oops, that slipped past me.